contracts/foundry.toml

```diff
@@ -5,4 +5,22 @@ libs = ['lib']
 solc_version = '0.8.18'
 test = 'tests'
 
-# See more config options https://github.com/foundry-rs/foundry/tree/master/config
\ No newline at end of file
+[fmt]
+line_length = 100
+tab_width = 4
+bracket_spacing = false
+int_types = 'preserve'
+multiline_func_header = 'params_first'
+quote_style = 'double'
+number_underscores = 'preserve'
+override_spacing = true
+wrap_comments = false
```

Comment:
Not 100% what the effects would be, but I think it may make sense to set this to `true` also.

Reply:
Yeah, I don't mind it in theory, but what it would do right now is just take any comment over `line_length` and put the extra on a newline, so in practice if we have formatted a bunch of comments ourselves, it may end up looking like 

`
// This comment used to be super long 
// but
// the formatter made it kind of ugly.
`

So someone would have to go through and manually clean then up.

Reply:
Ah yep, don't like that.